src/Monolog/Formatter/HtmlFormatter.php

```diff
@@ -71,9 +70,7 @@ protected function addRow(string $th, string $td = ' ', bool $escapeTd = true):
      */
     protected function addTitle(string $title, int $level): string
     {
-        $title = htmlspecialchars($title, ENT_NOQUOTES, 'UTF-8');
-
-        return '<h1 style="background: '.$this->logLevels[$level].';color: #ffffff;padding: 5px;" class="monolog-output">'.$title.'</h1>';
+        return "<h1 style=\"background: {$this->logLevels[$level]}; color: #ffffff;padding: 5px;\" class=\"monolog-output\">{$this->convertToSpecialChars($title)}</h1>";
```

Comment:
Frankly I'm not sure if having function calls within an interpolated string is what I would consider an improvement here..